docs/content/latest/reference/configuration/yb-master.md

```diff
@@ -381,7 +387,12 @@ yb-admin --master_addresses <master-addresses> --tablet_split_size_threshold_byt
 ```
 
 - *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
-- *bytes*: The threshold size, in bytes, before each tablet should be split. Default value is `0`, Default value of `0` disables dynamic tablet splitting.
+- *bytes*: The threshold size, in bytes, before each tablet should be split. Default value of `0` disables automatic tablet splitting.
```

Comment:
`after which tablets should be split`, not before

Reply:
good catch -- didn't deal as planned with whether it is after or at that size